prediction_market_agent/agents/think_thoroughly_agent/pinecone_utils.py

```diff
@@ -0,0 +1,22 @@
+import typing as t
+
+from prediction_market_agent_tooling.markets.omen.data_models import OmenMarket
+
+from prediction_market_agent.agents.think_thoroughly_agent.models import (
+    PineconeMetadata,
+)
+
+
+def create_texts_from_omen_markets(markets: list[OmenMarket]) -> list[str]:
+    return [m.question_title for m in markets]
```

Comment:
`create_texts` is a little misleading, should it create some special texts or just extract the titles?

Reply:
removed